help/en/html/doc/Technical/Help.shtml

```diff
@@ -203,18 +204,18 @@
 
         <p>The file name cannot contain an underscore character. The file suffix is .shtml.</p>
 
-        <p>The <strong>id</strong> attribute can assigned to HTML tags and then referenced by
+        <p>The <strong>id</strong> attribute can be assigned to HTML tags and then referenced by
         JmriHelp. This makes it possible to display window specific help information within a
         larger help document.</p>
 
         <p>Do not use spaces or <code>%20</code> in id names. If you must use a phrase for a name,
         use the underscore character: "This_Anchor", not "This Anchor.</p>
 
-        <p>When describing a sequence of user actions or pulldowns, use the double right arrow HTML
+        <p>When describing a sequence of user actions or pull downs, use the double right arrow HTML
         entity <strong>ampersand-rArr-semicolon</strong>, and label the whole sequence as "strong".
         For example:<br>
         <strong>File &rArr; Store Panels</strong><br>
-        A non-breaking space can be used before the right arrow to control line breaks.</p>
```

Comment:
"non-breaking" is the correct word, not "non-blanking".

Reply:
@icklesteve 
See: https://en.wikipedia.org/wiki/Non-breaking_space

Reply:
I thought it looked odd however not picked up in review of #11584  , reverted.